src/main/java/com/google/firebase/auth/FirebaseAuth.java

```diff
@@ -841,6 +919,65 @@ protected Void execute() throws FirebaseAuthException {
     };
   }
 
+  /**
+   * Deletes the users specified by the given identifiers.
+   *
+   * <p>Deleting a non-existing user won't generate an error. (i.e. this method is idempotent.)
+   * Non-existing users will be considered to be successfully deleted, and will therefore be counted
+   * in the DeleteUsersResult.getSuccessCount() value.
+   *
+   * <p>A maximum of 1000 identifiers may be supplied. If more than 1000 identifiers are
+   * supplied, this method throws an {@link IllegalArgumentException}.
+   *
+   * <p>This API is currently rate limited at the server to 1 QPS. If you exceed this, you may get a
```

Comment:
Suggest: 

"This API has a rate limit of 1 QPS. Exceeding the limit may result in a quota exceeded error. If you want to delete more than 1000 users, we suggest adding a delay to ensure you don't exceed the limit."

Reply:
Done.